pkgs/applications/networking/instant-messengers/rambox/bare.nix

```diff
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
     inherit src;
 
     nodejs = nodejs-8_x;
-    sha256 = "0hbw47653wh159c34f0rlj3p7xy0lvsyp0wh2hl35kv3fnsfbbm0";
+    sha256 = "0mg1ilz65qyxgh5zg1f8fhx7ygz10w8ifamflpwbk8mrvqpkjlvr";
```

Comment:
I got: 0hbw47653wh159c34f0rlj3p7xy0lvsyp0wh2hl35kv3fnsfbbm0

Reply:
Oh no, not good if those things happen: @yegortimoshenko

Reply:
In the long run we could switch to a mode like in: https://github.com/NixOS/nixpkgs/pull/44076

Reply:
Actually I also have 8.12.0 in my branch but still get `0hbw47653wh159c34f0rlj3p7xy0lvsyp0wh2hl35kv3fnsfbbm0`